tools/c7n_azure/c7n_azure/actions.py

```diff
@@ -12,16 +12,37 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 """
-Tag Actions to perform on Azure resources
+Actions to perform on Azure resources
 """
-from c7n.actions import BaseAction
+import datetime
+from azure.mgmt.resource.resources.models import GenericResource, ResourceGroupPatchable
+from msrestazure.azure_exceptions import CloudError
 from c7n import utils
+from c7n.actions import BaseAction
+from c7n_azure.provider import resources
 from c7n.filters import FilterValidationError
-from azure.mgmt.resource.resources.models import GenericResource, ResourceGroupPatchable
+
+
+def utcnow():
```

Comment:
Took me a minute to figure out what this was for :)

Reply:
Ya it seemed the cleanest way to target this single file. I didn't want to mock the entire datetime class because other parts of CC use it and I would need to implement every method they used.